Source/Core/Core/ConfigManager.h

```diff
@@ -304,6 +305,12 @@ struct SConfig
   int m_Volume;
   std::string sBackend;
 
+#ifdef _WIN32
+  // WSAPI settings
+  bool m_WASAPIExclusive;
+  std::string s_WASAPIDevice;
```

Comment:
`s_` should probably be `m_`, `s_` implies a static variable.

Reply:
It's ``sWASAPIDevice`` right now, like ``sBackend``. Or should it be ``m_Backend``?